awswrangler/_distributed.py

```diff
@@ -105,12 +108,20 @@ def register(cls, name: Optional[str] = None) -> None:
     @classmethod
     def initialize(cls, name: Optional[str] = None) -> None:
         """Initialize the distribution engine."""
-        engine_name = cls.get_installed().value if not name else name
+        engine_name = name or cls.get_installed().value
```

Comment:
Very minor but can we pass class here, do `.name` and avoid having to do `.value` when initialising:
```
wr.engine.initialize(EngineEnum.PYTHON.value)
```